components/Tabs/Tabs.js

```diff
@@ -1,62 +1,83 @@
 
 class TabLink {
-  constructor(element) {
+  constructor(tab, link) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+     this.link = link;
+  //  console.log(this.link);
     // Get the custom data attribute on the Link
-    // this.data;
-    
+     this.data = this.link.dataset.tab;
+    //console.log(this.data);
+
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+
+     this.itemElement = document.querySelector(`.tabs-item[data-tab="${tab, this.data}"]`);
+     //console.log(this.itemElement)
+
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
+     this.tabItem = new TabItem(tab, this.itemElement);
+      //console.log(this.tabItem)
     // Add a click event listener on this instance, calling the select method on click
-
+     this.link.addEventListener('click', this.select.bind(this));
   };
 
   select() {
     // Get all of the elements with the tabs-link class
-    // const links;
+     const links = document.querySelectorAll('.tabs-link');
 
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
+     Array.from(links).forEach(link => {
+
+        link.classList.remove('tabs-link-selected');
+      // console.log('testing')
+     });
 
     // Add a class named "tabs-link-selected" to this link
-    // this.element;
-    
-    // Call the select method on the item associated with this link
+     this.link.classList.toggle('tabs-link-selected');
 
+    // Call the select method on the item associated with this link
+     this.tabItem.select();
   }
 }
 
 class TabItem {
-  constructor(element) {
+  constructor(tab, data) {
     // Assign this.element to the passed in element
-    // this.element;
+     this.element = data;
+     this.tab = tab;
+
   }
 
   select() {
     // Select all ".tabs-item" elements from the DOM
-    // const items;
+    const items = Array.from(document.querySelectorAll('.tabs-item'));
 
     // Remove the class "tabs-item-selected" from each element
-    
+     items.forEach(item => {
+       item.classList.remove('tabs-item-selected');
+     })
+
     // Add a class named "tabs-item-selected" to this element
-    //this.element;
+     this.element.classList.toggle('tabs-item-selected');
```

Comment:
The result would be the same either way, but I'd recommend using `.classList.add` here to have more declarative code (since the `toggle` method can also remove a class, whereas there will be no mistaking how the `add` method always adds a class).